src/Examples/Advanced/CustomUnitModels/methanol_param_VLE.py

```diff
@@ -119,43 +114,37 @@ def build(self):
                                      ('H2', 'C'): 3.19,
                                      ('CH3OH', 'A'): 18.5875,
                                      ('CH3OH', 'B'): 3626.55,
-                                     ('CH3OH', 'C'): -34.29}
+                                     ('CH3OH', 'C'): -34.29} # these Antoine coefficients assume mmHG and K
 
         Cp = self.config.Cp
-        Cv = Cp - self.gas_constant.value
+        Cv = Cp - Constants.gas_constant/1000/pyunits.mol*pyunits.kmol # convert gas constant from J/mol-K to MJ/kmol-K
```

Comment:
You should use `pyunits.convert()` here, rather than hard coding the conversion.